src/utils/network/session-core.js

```diff
@@ -383,35 +383,37 @@ export const createDocumentContainer = async (session, podUrl) => {
 };
 
 /*
-  User Inbox Section
-
-  Functions here deal primarily with user inbox/outbox on PASS
+  User Message Section
+  
+  Functions here deal primarily with user messages on PASS
 */
 
 /**
  * Function that gets list of inbox TTL file messages and returns the messages as
  * JSON
```

Comment:
Nitpick, but this function returns an object, not JSON. JSON is a string.

Reply:
Gotcha. Switching to just object.